[Bug]: Postgres Bulk Loader throws class exception

### Apache Hop version?

2.4-GA

### Java version?

11

### Operating system

Linux

### What happened?

Take a fresh copy of 2.4-GA.  Take any data to the PG Bulk Loader against any PG table and it will throw the following exception.


2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - class org.postgresql.jdbc.PgConnection cannot be cast to class org.postgresql.PGConnection (org.postgresql.jdbc.PgConnection is in unnamed module of loader org.apache.hop.core.plugins.HopURLClassLoader @3eb64e63; org.postgresql.PGConnection is in unnamed module of loader org.apache.hop.core.plugins.HopURLClassLoader @72155315)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 	at org.apache.hop.pipeline.transforms.pgbulkloader.PGBulkLoader.doCopy(PGBulkLoader.java:161)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 	at org.apache.hop.pipeline.transforms.pgbulkloader.PGBulkLoader.processRow(PGBulkLoader.java:240)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 	at org.apache.hop.pipeline.transform.RunThread.run(RunThread.java:55)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 	at java.base/java.lang.Thread.run(Thread.java:829)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - Caused by: java.lang.ClassCastException: class org.postgresql.jdbc.PgConnection cannot be cast to class org.postgresql.PGConnection (org.postgresql.jdbc.PgConnection is in unnamed module of loader org.apache.hop.core.plugins.HopURLClassLoader @3eb64e63; org.postgresql.PGConnection is in unnamed module of loader org.apache.hop.core.plugins.HopURLClassLoader @72155315)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 	at org.apache.hop.pipeline.transforms.pgbulkloader.PGBulkLoader.doCopy(PGBulkLoader.java:158)
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - 	... 3 more
2023/05/09 02:12:36 - PostgreSQL Bulk Loader.0 - Finished processing (I=0, O=0, R=1, W=0, U=0, E=1)


### Issue Priority

Priority: 2

### Issue Component

Component: Transforms

--- a/plugins/databases/postgresql/src/main/java/org/apache/hop/databases/postgresql/PostgreSqlDatabaseMeta.java
+++ b/plugins/databases/postgresql/src/main/java/org/apache/hop/databases/postgresql/PostgreSqlDatabaseMeta.java
@@ -28,7 +28,7 @@ import org.apache.hop.core.row.IValueMeta;
 /**
  * Contains PostgreSQL specific information through static final members
  */
-@DatabaseMetaPlugin(type = "POSTGRESQL", typeDescription = "PostgreSQL", documentationUrl = "/database/databases/postgresql.html")
+@DatabaseMetaPlugin(type = "POSTGRESQL", typeDescription = "PostgreSQL", documentationUrl = "/database/databases/postgresql.html", classLoaderGroup = "postgres-db")
 @GuiPlugin(id = "GUI-PostgreSQLDatabaseMeta")
 public class PostgreSqlDatabaseMeta extends BaseDatabaseMeta implements IDatabase {
   private static final int GB_LIMIT = 1_073_741_824;

--- a/plugins/transforms/pgbulkloader/src/main/java/org/apache/hop/pipeline/transforms/pgbulkloader/PGBulkLoaderMeta.java
+++ b/plugins/transforms/pgbulkloader/src/main/java/org/apache/hop/pipeline/transforms/pgbulkloader/PGBulkLoaderMeta.java
@@ -50,7 +50,8 @@ import java.util.List;
     name = "i18n::PGBulkLoader.Name",
     categoryDescription = "i18n:org.apache.hop.pipeline.transform:BaseTransform.Category.Bulk",
     keywords = "i18n::PGBulkLoaderMeta.keyword",
-    documentationUrl = "/pipeline/transforms/postgresbulkloader.html")
+    documentationUrl = "/pipeline/transforms/postgresbulkloader.html",
+    classLoaderGroup = "postgres-db")
 public class PGBulkLoaderMeta extends BaseTransformMeta<PGBulkLoader, PGBulkLoaderData> implements
         IProvidesDatabaseConnectionInformation {
 

